engine/dlib/src/wscript

```diff
@@ -313,6 +313,12 @@ def build(bld):
         if new_opt_level:
             basis_encoder.env.append_unique('CXXFLAGS', ['-Os'])
 
+        basis_encoder_noasan = bld.stlib(features = basis_encoder.features + ['skip_asan'],
```

Comment:
Asan and python/Java is quite inconvenient, so it's sometimes better to just avoid the problem, e.g. making texc use non_asan libraries